doc/gocc_user_guide.tex

```diff
@@ -121,7 +121,7 @@ \section{Getting started}
 
 					or
 
-				\item Alternatively clone the repository: \url{https://github.com/goccmack/gocc/source/checkout}. Followed by:
+				\item Alternatively clone the repository: \url{https://github.com/goccmack/gocc/archive/master.zip}. Followed by:
```

Comment:
Sure, makes sense. I'll strip it and replace with https://github.com/goccmack/gocc